common/testData/localeIdentifiers/likelySubtags.txt

```diff
@@ -0,0 +1,1576 @@
+# Test data for Likely Subtags
+#  Copyright © 1991-2023 Unicode, Inc.
+#  For terms of use, see http://www.unicode.org/copyright.html
+#  SPDX-License-Identifier: Unicode-DFS-2016
+#  CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+#
+# Test data for https://www.unicode.org/reports/tr35/tr35.html#Likely_Subtags
+#
+# Format:
+
+# Source ;	AddLikely ;	RemoveFavorScript ;	RemoveFavorRegion
+#   Source: a locale to which the following operations are applied.
+#   AddLikely: the result of the Add Likely Subtags.
+#   RemoveFavorScript: Remove Likely Subtags, when the script is favored.
+#                      Only included when different than AddLikely.
+#   RemoveFavorRegion: Remove Likely Subtags, when the region is favored.
+#                      Only included when different than RemoveFavorScript.
```

Comment:
Observation: I'm not sure we implement `RemoveFavorScript` in ICU4X. Good to have this test data to find a deficiency in our feature coverage that we can work toward fixing. :)